.github/workflows/ci_frontend.yml

```diff
@@ -50,3 +50,44 @@ jobs:
           path: reports/jest-*.xml
           reporter: jest-junit
           working-directory: 'RetrospectiveExtension.Frontend'
+        
+      - if: steps.changes.outputs.src == 'true'
+        name: Get Coverage for badge
+        working-directory: 'RetrospectiveExtension.Frontend'
+        run: |
+          SUMMARY="$(npm test -- --coverageReporters='text-summary' | tail -2 | head -1)"
+          TOKENS=($SUMMARY)
+          # process.env.COVERAGE = '31.93%';
+          echo "COVERAGE=$(echo ${TOKENS[2]})" >> $GITHUB_ENV
+          # var REF = 'refs/pull/27/merge.json';
+          REF=${{ github.ref }}
+          # console.log('github.ref: ' + REF);
+          echo "github.ref: $REF"
+          # var PATHS = REF.split('/');
+          IFS='/' read -ra PATHS <<< "$REF"
+          # var BRANCH_NAME = PATHS[1] + PATHS[2];
+          BRANCH_NAME="${PATHS[1]}_${PATHS[2]}"
+          # console.log(BRANCH_NAME); // 'pull_27'
+          echo $BRANCH_NAME
+          # process.env.BRANCH = 'pull_27';
+          echo "BRANCH=$(echo ${BRANCH_NAME})" >> $GITHUB_ENV
+
+      - if: steps.changes.outputs.src == 'true'
+        name: Create the Badge
+        uses: schneegans/dynamic-badges-action@v1.0.0
+        with:
+          auth: ${{ secrets.GIST_SECRET }}
+          gistID: b414faf6b91eaaa6ec7d82882be36935
```

Comment:
Is it possible to make this a secret ? I know it is part of the URL in the following locations:
https://github.com/NarmathaBala/vsts-extension-retrospectives/pull/63/files#diff-50fc973073497e2e2724ebaf4b3b5511ca0b5f07f2693e58963f7017fa11332eR91

https://github.com/NarmathaBala/vsts-extension-retrospectives/pull/63/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3


But I'm wondering if it is possible to do something similar with these URLs ... it seems like it would be better for future proofing to have one place where the `gistID` can be updated

Reply:
I read that part of the description and then immediately forgot when reviewing the actual code... 🤦